backend/src/schema/types/User.js

```diff
@@ -0,0 +1,7 @@
+const { gql } = require("apollo-server");
+
+module.exports.AuthorType = gql`
+  type User {
+    name: String
```

Comment:
You could make the `name` the `ID`:
```suggestion
    name: ID
```

Reply:
what's the difference? its not like there are any primary keys or something like in sql?

Reply:
`ID` is a special scalar type: https://graphql.org/learn/schema/#scalar-types

GraphQL clients can update their cache if they fetch the object with the ID. That's why you should respond with the updated object when you `UpdateTodo` or something.

Imagine that you have a reactive frontend framework like VueJS or React. All you have to do is to send a mutation to the backend. Your frontend app will automagically re-render the HTML where your object is used.

Reply:
hm I see. Than I'll change it